apps/omg_eth/lib/eth.ex

```diff
@@ -249,4 +272,16 @@ defmodule OMG.Eth do
     result
     |> Map.put(:eth_height, int_from_hex(eth_height))
   end
+
+  defp get_signer_passphrase("0x00a329c0648769a73afac7f9381e08fb43dbea72") do
+    # Parity coinbase address in dev mode, passphrase is empty
+    {:ok, ""}
+  end
+
+  defp get_signer_passphrase(_) do
+    case System.get_env("SIGNER_PASSPHRASE") do
+      nil -> {:error, :please_provide_passphrase_unlocking_ethereum_account_managed_by_parity}
```

Comment:
how about `{:error, :passphrase_missing}` and `Logger.error("the longer message")`?